MetaMask/metamask-extension
# feat: codefence Account Watcher for flask

## **Description**

This PR puts the Account Watcher feature under the Flask codefence, restricting it to only be included in Flask builds.

## **Manual testing steps**

1. Build the extension in Flask mode
2. Verify the account watcher feature is present
3. Build the extension in non-Flask mode 
4. Verify the account watcher feature is not present

## **Screenshots/Recordings**

The option to "Watch Ethereum Accounts (Beta)" should **NOT** be present on the _Main_ build

https://github.com/user-attachments/assets/e9cd21fa-e8fb-4c63-b888-48178df8730a

The option to "Watch Ethereum Accounts (Beta)" should **BE PRESENT** on the _Flask_ build

https://github.com/user-attachments/assets/b641141e-878e-4f96-b41b-f14745de6ced

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
